sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_version.py

```diff
@@ -4,4 +4,4 @@
 # license information.
 # --------------------------------------------------------------------------
 
-VERSION = "12.0.3"
+VERSION = "12.1.0"
```

Comment:
I will make it 12.1.0b, let me know if you feel it's weird, I just feel not confident about Jumbo blob feature, since with max_single_put_size equals 64MB, user always get timeout .... I feel panic about max_single_put_size equals 5GB